### PR TITLE
refactor : migrate chain specific overview page

### PR DESCRIPTION
**Pull request by Teja2045**
#459

### CODE REVIEW

File: frontend/src/pages/Overview.js [hunk 1]
```diff
@@ -19,32 +19,42 @@ import { useNavigate } from "react-router-dom";
...
-  const dispatch = useDispatch();
-
-  const { config } = chainInfo;
-  const coinDecimals = config?.currencies[0].coinDecimals || 1;
-  const coinDenom = config?.currencies[0].coinMinimalDenom || "";
-
   useEffect(() => {
     if (connected && config.currencies.length > 0) {
-      setTotalBalance(parseBalance([balance.balance], coinDecimals, coinDenom));
+      if (balance?.[chainID]?.list?.[0] !== undefined)
```
Comment (aleem1314) [inline frontend/src/pages/Overview.js:53]: The main denomination can be located anywhere in the array. To obtain the balance of the main denomination, you can either use the "balance by denom" API or iterate over the balances array to find the balance associated with the main denomination.

Comment (Teja2045) [reply frontend/src/pages/Overview.js]: done

File: frontend/src/features/bank/bankSlice.js [hunk 2]
```diff
@@ -142,7 +142,14 @@ export const bankSlice = createSlice({
           state.balances[chainID] = result;
         }
       })
-      .addCase(getBalances.rejected, (state, action) => {});
+      .addCase(getBalances.rejected, (state, action) => {
+        const chainID = action.meta.arg.chainID;
+        state.balances[chainID] = {
+          status: "idle",
+          errMsg: "",
```
Comment (aleem1314) [inline frontend/src/features/bank/bankSlice.js:149]: Lets store error message

File: server/logs.txt
Comment (aleem1314) [inline server/logs.txt:1]: delete this file

**Review by aleem1314 — APPROVED** [review]
Pre approving

**Pull request merged**